cypress.config.ts

```diff
@@ -25,6 +26,8 @@ export default defineConfig({
       on('task', {
         reset: resetStubs,
 
+        resetRedis: resetRedisDb,
```

Comment:
I'd have been tempted to link this to the `reset` task so you don't need to remember to add it in as a separate call for each test, like we do for [use of force](https://github.com/ministryofjustice/use-of-force/blob/main/cypress.config.ts#L32)

Reply:
agreed